File: src/WebApiHelpPage/Areas/HelpPage/ModelDescriptions/ComplexTypeModelDescription.cs
using System.Collections.Generic;

namespace ROOT_PROJECT_NAMESPACE.Areas.HelpPage.ModelDescriptions
{
    public class ComplexTypeModelDescription : ModelDescription
    {
        public ComplexTypeModelDescription()
        {
            Properties = new List<PropertyDescription>();
        }

        public IList<PropertyDescription> Properties { get; private set; }
    }
}

File: src/WebApiHelpPage/VB/Areas/HelpPage/ModelDescriptions/ComplexTypeModelDescription.vb
Imports System.Collections.Generic

Namespace Areas.HelpPage.ModelDescriptions
    Public Class ComplexTypeModelDescription
        Inherits ModelDescription
        Private _properties As IList(Of PropertyDescription)

        Public Sub New()
            Properties = New List(Of PropertyDescription)()
        End Sub

        Public Property Properties() As IList(Of PropertyDescription)
            Get
                Return _properties
            End Get
            Private Set(value As IList(Of PropertyDescription))
                _properties = value
            End Set
        End Property
    End Class
End Namespace